src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -127,11 +175,9 @@ We support
 
             */
 
-            var enabledFor = new List<FeatureFilterConfiguration>();
+            Debug.Assert(configurationSection != null);
```

Comment:
>Debug.Assert(configurationSection != null); [](http://example.com/codeflow?start=12&length=43)

Should we throw an exception instead of assert?

Reply:
It's a private method so throwing here wouldn't be expected.